specification/iothub/resource-manager/Microsoft.Devices/2017-07-01/iothub.json

```diff
@@ -1013,9 +1013,472 @@
         },
         "deprecated": false
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/certificates": {
+      "get": {
+        "tags": [ "Certificates" ],
+        "summary": "Get the certificate list.",
+        "description": "Returns the list of certificates.",
+        "operationId": "Certificates_ListByIoTHub",
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The name of the resource group that contains the IoT hub.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "description": "The name of the IoT hub.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The body contains all the certificate list.",
+            "schema": {
+              "$ref": "#/definitions/CertificateListDescription"
```

Comment:
[1]. Is this a paged collection? i.e. does service return all the certificates in a single response or page by page with a page containing link to fetch next page? see https://github.com/Azure/azure-rest-api-specs/blob/current/documentation/swagger-extensions.md#x-ms-pageable for more on paged collection. 

[2]. Is there any limit on number of certificates that can be created under an IoT hub resource?